### DISCUSSION TITLE
功能建议：启动时默认最小化和启动游戏

### DISCUSSION THREAD

**Discussion by ningbowang** [cangzhang/ChampR#166] [General]
添加一项设置，可以选择默认最小化运行。
![Snipaste_2021-01-20_17-12-47.png](https://i.loli.net/2021/01/20/Fo12KqkSmGvBdHW.png)

如果能在CharmpR中启动游戏，就真的太完美了。当然也担心替代了wegame被官方盯上，导致目前的版本都不能再开放了（当前可以通过**[批处理文件](https://github.com/cangzhang/champ-r/issues/36)**同时打开两个软件）

**Comment by cangzhang** [comment]
感谢反馈，我们会在之后的版本加上「启动时最小化」选项
